pinot-core/src/test/java/org/apache/pinot/core/transport/QueryRoutingTest.java

```diff
@@ -54,6 +54,7 @@
 import static org.testng.Assert.assertNotNull;
 import static org.testng.Assert.assertNull;
 import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertFalse;
```

Comment:
This is causing the Spotless plugin check to fail due to incorrect import ordering